src/shared/components/ShareWithColleagues/ShareWithColleagues.helpers.tsx

```diff
@@ -0,0 +1,12 @@
+import { flatten, groupBy } from 'lodash';
```

Comment:
always import from 'lodash-es' since it supports tree shaking and makes the final javascript file bundle smaller